### PR TITLE
fix https://github.com/Hackerpilot/Dscanner/issues/251

### PR DESCRIPTION
**Pull request by timotheecour**

### CODE REVIEW

File: src/main.d [hunk 1]
```diff
@@ -274,18 +274,29 @@ int run(string[] args)
...
 
+
 string[] expandArgs(string[] args)
 {
+	// isFile can throw if it's a broken symlink.
+	bool isFileSafe(T)(T a){
+		try{
+			return isFile(a);
+		}
+		catch{
```
Comment (Hackerpilot) [inline src/main.d]: You want to catch Throwable here?

File: src/main.d [hunk 2]
```diff
@@ -274,18 +274,29 @@ int run(string[] args)
 	return 0;
 }
 
+
 string[] expandArgs(string[] args)
 {
+	// isFile can throw if it's a broken symlink.
+	bool isFileSafe(T)(T a){
```
Comment (Hackerpilot) [inline src/main.d]: Please put opening braces on the next line the way it's done in the rest of the project.

**Comment by timotheecour** [comment]
PTAL

**Comment by Hackerpilot** [comment]
I have no idea what PTAL means.

**Comment by timotheecour** [comment]
Please Take Another Look
common in code reviews :)

**Pull request merged**